src/main/api/2/studio-api-2.yaml

```diff
@@ -1100,6 +1100,89 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /users/me/properties:
+    get:
+      tags:
+        - users
+      summary: Retrieve properties for the current user
+      description: 'Required Permission: "LOGGED_IN"'
+      operationId: getUserProperties
+      parameters:
+        - name: siteId
+          schema:
+            type: string
+          in: query
+          description: The id of the site (defaults to global)
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  properties:
+                    type: object
+                    additionalProperties:
+                      type: string
+                    description: The properties of the current user
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+    post:
+      tags:
+        - users
+      summary: Update properties for the current user
+      description: 'Required Permission: "LOGGED_IN"'
+      operationId: updateUserProperties
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                siteId:
+                  type: string
+                  description: The id of the site (defaults to global)
+                upsert:
+                  type: object
+                  additionalProperties:
+                    type : string
+                  description: Properties to update or add
+                delete:
+                  type: array
+                  items:
+                    type: string
+                  description: List of keys to delete
```

Comment:
Perhaps here we could have the siteId be sent as part of the objects sent to upsert/delete so we could CRUD properties across sites in a single request?

Reply:
+1

Reply:
As I mentioned here https://github.com/craftercms/studio/pull/2295#discussion_r533432653 this would be the first cross-site operation we implement so I think we need to discuss the design further to make sure it doesn't have conflict with the existing APIs

Reply:
It feels clear given that the site is is moved into each object.